cmd/gophermart/main.go

```diff
@@ -1,3 +1,138 @@
 package main
 
-func main() {}
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/AleksandrGurkin/gophermart/config"
+	"github.com/AleksandrGurkin/gophermart/internal/handlers"
+	"github.com/AleksandrGurkin/gophermart/internal/services"
+	"github.com/AleksandrGurkin/gophermart/internal/storage/orders"
+	postgres2 "github.com/AleksandrGurkin/gophermart/internal/storage/orders/postgres"
+	"github.com/AleksandrGurkin/gophermart/internal/storage/sessions"
+	"github.com/AleksandrGurkin/gophermart/internal/storage/sessions/memory"
+	"github.com/AleksandrGurkin/gophermart/internal/storage/users"
+	"github.com/AleksandrGurkin/gophermart/internal/storage/users/postgres"
+	mdl "github.com/AleksandrGurkin/gophermart/middleware"
+	"github.com/AleksandrGurkin/gophermart/migrations"
+	http2 "github.com/AleksandrGurkin/gophermart/provider/accrual/http"
+	"github.com/AleksandrGurkin/xlog"
+	"github.com/AleksandrGurkin/xlog/xzerolog"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+)
+
+func main() {
+	//init config
+	cfg, err := config.Read("./config.yaml")
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	//init logger
+	logger, err := xzerolog.NewXZerolog(xlog.LoggerCfg{
+		Level: cfg.LogLevel,
+		Out:   os.Stdout,
+	})
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	//migrate scheme
+	err = migrations.Migrate(cfg.DatabaseURL, cfg.MigrationDir)
+	if err != nil {
+		logger.Error(err.Error())
+		return
+	} else {
+		logger.Info("Migration done.")
+	}
```

Comment:
```suggestion
	} 
	logger.Info("Migration done.")
```
можно без else писать, если  err != nil то программа не пойдет дальше